locale/core.yml

```diff
@@ -217,6 +219,41 @@ core:
       remove_button: => core.ref.remove
       upload_button: Choose an Image...
 
+    users:
+      title: => core.ref.users
+      description: A paginated list of all users on your forum.
+
+      total_users: "Total users: {count}"
+
+      grid:
+        invalid_column_content: Invalid
+
+        default_columns:
+          id: ID
+
+          username: => core.ref.username
+          profile_link_tooltip: Visit {username}'s profile
+
+          join_time: Joined
+
+          group_badges: Groups
+          group_badges_none: None
+
+          email: Email
+          email_visibility_show: Show email address
+          email_visibility_hide: Hide email address
+          # Fake email address which is blurred before being replaced by the real email
+          email_hidden: "abcdefg12345@gmail.com"
```

Comment:
This is out of order, should be alphabetic. Also applies to other translations in this file.